afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+lei seg	-2
+trist	-2
+dessverre	-2
+trygt	1
+sikrere	2
+sikkerhet	1
+honnør	2
+hilste	2
+salutter	2
+frelse	2
+sarkastisk	-2
+lagre	2
+lagret	2
+sparing	1
+skandale	-3
+skandaløs	-3
+skandalene	-3
+syndebukk	-2
+syndebukker	-2
+arr	-2
+skummelt	-2
+skeptisk	-2
+skjenne	-2
+kule	3
+hånlig	-2
+hyle	-2
+skrek	-2
+skrudd	-2
+feilet	-3
+avskum	-3
+sømløs	2
+sømløst	2
+sikre	2
+sikret	2
+oppvigleri	-2
+opprørsk	-2
+forført	-1
+selvbesmittelse	-2
+selvsikkerhet	2
+selvmotsigende	-2
+selv villedet	-2
+selvopptatt	-3
+egoisme	-3
+setning	-2
+setninger	-2
+utmålingen	-2
+bosetting	1
+bosetninger	1
+alvorlig	-2
+kjønnsdiskriminerende	-2
+skjelvende	-2
+skam	-2
+skjemmes	-2
+skammelig	-2
+dele	1
+delt	1
+aksjer	1
+sjokk	-2
+sjokkert	-2
+sjokkerende	-2
+støt	-2
+skyte	-1
+nærsynt	-2
+nærsynthet	-2
+spissmus	-4
+sjenert	-2
+syk	-2
+bivirkning	-2
+bivirkninger	-2
+sukk	-2
+betydelige	1
+lyddemping	-1
+enkelhet	1
+oppriktig	2
+vennlig hilsen	2
+since	2
+oppriktighet	2
+syndig	-3
+ensporet	-2
+synder	-2
+skeptiker	-2
+skepsis	-2
+skeptikere	-2
+skråstrek	-2
+kuttet	-2
+skråstreker	-2
+slaveri	-3
+glatt	2
+stiligere	2
+slurvet	-2
+treg	-2
+ludder	-5
+smartere	2
+smarteste	2
+smøre	-2
+smil	2
+smilte	2
+smiler	2
+smilende	2
+smugle	-2
+smuglet	-2
+smugling	-2
+smugler	-2
+sleipe	-1
+nyse	-2
+nøs	-2
+nyser	-2
+nysing	-2
+trykkrør	-2
+tankevekkende	1
+høytidelig	-1
+fast	2
+solidaritet	2
+størknet	2
+stivner	2
+størkne	2
+løsning	1
+løsninger	1
+en type	0
+son-of-a-tispe	-5
+sofistikert	2
+sår	-1
+sorgfull	-2
+beklager	-1
+romslig	1
+spammere	-3
+gnist	1
+gnisten	3
+glitrer	3
+musserende	3
+spydspiss	2
+spekulativ	-2
+ånd	1
+spirit	-2
+ondskapsfull	-2
+bortskjemte	-2
+plettfrie	2
+struttende	2
+sløse	-2
+sløst bort	-2
+sløsing	-2
+stikke	-2
+knivstukket	-2
+stabil	2
+stoppet	-1
+steile	-2
+utholdenhet	2
+stinket	-2
+oppsiktsvekkende	3
+sulte	-2
+sultet	-2
+sulter	-2
+standhaftig	2
+stjele	-2
+stjeler	-2
+stereotype	-2
+kvalt	-2
+stimulere	1
+stimulert	1
+stimulerer	1
+stimulerende	2
+stikkende	-2
+stinke	-2
+stinkende	-2
+stinker	-2
+stjal	-2
+stjålet	-2
+stoppe	-1
+stoppe	-1
+stopp	-1
+traust	2
+rett	1
+rar	-1
+styrke	2
+forsterke	2
+styrking	2
+styrker	2
+understreke	-1
+stresset	-2
+streik	-1
+sterk	2
+sterkere	2
+sterkeste	2
+streve	-2
+slet	-2
+sliter	-2
+sittende fast	-2
+slående	4
+dumhet	-3
+beleven	2
+stevning	-2
+betydelig	1
+i det vesentlige	1
+undergravende	-2
+lykkes	3
+lyktes	3
+suksess	2
+vellykket	3
+suge	-3
+suger	-3
+saksøke	-2
+saksøkt	-2
+sues	-2
+lide	-2
+led	-2
+lidende	-2
+lider	-2
+selvmord	-2
+velegnet	2
+mutt	-2
+solskinn	2
+suveren	5
+overlegen	2
+brukerstøtte	2
+støttet	2
+støttespillere	1
+bære	1
+støttende	2
+støtter	2
+overlevde	2
+overlevende	2
+mistenkt	-1
+mistanke	-1
+mistenkte	-1
+henge	-1
+suspendert	-1
+mistenkelig	-2
+bærekraft	1
+bærekraftig	2
+sverge	-2
+banning	-2
+sverger	-2
+søtere	3
+søteste	3
+fort	2
+raskt	2
+svindle	-3
+svindlerier	-3
+sympati	2
+plett	-2
+skjemt	-2
+tard	-2
+anløpe	-2
+anløpet	-2
+oksiderer	-2
```

Comment:
This word should probably have no sentiment.

Reply:
added to stoplist